### PR TITLE
metadata-scheme

### PR DESCRIPTION
**Pull request by Asbjoedt**
Hi Carl, Remco and Kaido

This one is a major PR. It makes an attempt at considering which metadata should be preserved under which circumstances and how can this be done.

This PR is one of my more rough and sketched out onces. I think it still needs changes, but at the same time I think getting it in early is better than not having it because it answers several of our discussions and references in the spec about how to document potential data loss either by choice of file format or by active removal of certain data.

I hope you will take kindly to this one.

### CODE REVIEW

File: Draft v1.0/Specification.md [hunk 1]
```diff
@@ -238,9 +244,39 @@ The following table gives guidelines for how to comply with the requirements in
...
+| ODS_4, OOXML_5 | External references | Any external reference to data e.g. data source, cell reference, RealTimeData. | String(s)<br><br>Cell position attribute | If removed |
+| ODS_5, OOXML_6 | Embedded objects | Any embedded object that has been removed. | String(s)<br><br>Cell position attribute | If removed |
+| STATIC_6 | Formulas | Any formulas that has been removed from the spreadsheet. | String(s)<br><br>Cell position attribute | Every time |
+| ODS_7, OOXML_8 | Macros | Any macros that has been removed from the spreadsheet. | String(s) | If removed |
+| ODS_11, OOXML_13, STATIC_6 | Cell hyperlinks | Any hyperlinks to websites in any cells. | String(s)<br><br>Cell position attribute | Every time |
+
+### 6.2 Example Metadata File
+You may find an example of a metadata preservation sidecar file created in XML [here](/v1.0/Example%20Metadata%20File.xml).
+
+## 7 Example Spreadsheets Software
```
Comment (carlwilson) [inline Draft v1.0/Specification.md:279]: The more I think about this, the less I'm convinced it's a great idea:

1. It just becomes the half-way house for things that there seems to be a desire to remove from the document.
2. We're in danger of inventing another metadata format where others exist for this kind of thing:
  - PREMIS for preservation events, and what was removed
  - various metadata formats for creator, title, etc.
  - METS as an information package standard to capture structural info, such as the relationships between the original sheet structure and CSV file (structMap)
  - ODF/OO XML for spreadsheet artefacts like macros and formulas
3. Remember this software doesn't redact files, it validates against the original specification and then enforces compliance with further rules/reports preservation risks.

Comment (Asbjoedt) [reply Draft v1.0/Specification.md:279]: Your validator is one piece of the software required to make this specification function e.g. the specification also has policies about OOXML and we don't create a validator for that format now.

We may consider removing the Metadata Example File (the sidecar) and solely make recommendations of which metadata to preserve in the context of spreadsheets and under which circumstances. Then we may write it is up to the organisation to initiate measures on how to capture the metadata.

**Comment by Asbjoedt** [comment]
This pull request has no been severely downscoped to only include recommendations on which metadata to preserve and under which circumstances. The technology used to capture and store the metadata should instead be decided by the organisation.

File: Draft v1.0/Specification.md (hunk 1 above)
Comment (RvanVeenendaal) [reply Draft v1.0/Specification.md:277]: From the perspective of significant properties, I would like to propose the following:

Spreadsheets can contain metadata such as (original) filename, title, author and  creation date, or have associated metadata about the context of or events related to the spreadsheet, Spreadsheet metadata SHOULD BE preserved when considered a significant property. It is up to your organization to determine the best approach and metadata standard for preserving such metadata. Metadata that is considered an artifact MAY BE removed.

**Review by RvanVeenendaal — CHANGES_REQUESTED** [review]

**Comment by Asbjoedt** [comment]
Kaido and Asbjørn looked at Remcos proposed text and adopted it in the pull request. We think the PR can be merged now.

**Review by kivilaan — APPROVED** [review]

**Review by RvanVeenendaal — APPROVED** [review]

**Review by RvanVeenendaal — APPROVED** [review]

**Pull request merged**